lib/phubme/comment_parser.ex

```diff
@@ -1,32 +1,15 @@
 defmodule PhubMe.CommentParser do
   def process_comment(body_params) do
     comment = get_in(body_params, ["comment", "body"])
-    sender  = get_in(body_params, ["comment", "user", "login"])
-    comment_source = get_in(body_params, ["comment", "html_url"])
-    IO.puts "Processing comment : \"" <> comment <> "\" from " <> sender
-    comment = comment
-              |> nicknames_present
-              |> extract_nicknames
-    case comment do
-      {:ok, full_comment, nicknames} -> {full_comment, nicknames, sender, comment_source}
-      {:error, _} -> "stop here"
-    end
+    sender = get_in(body_params, ["comment", "user", "login"])
+    source = get_in(body_params, ["comment", "html_url"])
+    nicknames = comment |> extract_nicknames
+    IO.puts "Processing comment : \"#{comment}\" from #{sender}"
```

Comment:
I have no idea if this interpolation syntax is faster that the concatenation that was here before. If I had to answer, my guess would be that it is faster.